pkg/grapicmd/context.go

```diff
@@ -129,8 +121,8 @@ var CtxSet = wire.NewSet(
 func ProvideFS(c *Ctx) afero.Fs                 { return c.FS }
 func ProvideViper(c *Ctx) *viper.Viper          { return c.Viper }
 func ProvideExecer(c *Ctx) exec.Interface       { return c.Execer }
-func ProvideIO(c *Ctx) *cli.IO                  { return c.IO }
+func ProvideIO(c *Ctx) clib.IO                  { return c.IO }
 func ProvideRootDir(c *Ctx) cli.RootDir         { return c.RootDir }
 func ProvideConfig(c *Ctx) *Config              { return &c.Config }
-func ProvideBuildConfig(c *Ctx) *BuildConfig    { return &c.Build }
+func ProvideBuildConfig(c *Ctx) *clib.Build     { return &c.Build }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideBuildConfig should have comment or be unexported